docs/project-overview/code-of-conduct.md

```diff
@@ -0,0 +1,34 @@
+---
+description: 'Be nice to one another.'
+---
+
+# Code of Conduct
+
+Airbyte's Slack community is growing incredibly fast. We're home to over 1500 data professionals and are growing at an awesome pace. We are proud of our community, and have provided these guidelines to support new members in maintaining the wholesome spirit we have developed here. We appreciate your continued commitment to making this a community we are all excited to be a part of.
+
+## Rule 1: Be respectful.
+
+## Rule 2: Check off your resolved questions.
+
+If you asked for help and received a useful reply, please place a ✅ reaction by your original post to mark it as resolved.
+
+## Rule 3: Don’t double-post.
+
+If you ask a question and don’t get a response, review your question for clarity and revise it instead of reposting it. If you still feel that you haven’t received an adequate response, feel free to ping @support.
+
+## Rule 4: Keep it public.
+
+This is a public forum; do not contact individual members of this community without their explicit permission, independent of reason.
+
+## Rule 5: No soliciting!
+If you’re a vendor, you may advertise your product in #shameless-plugs. Advertising your product anywhere else, especially in DMs, is strictly against the rules. We are appreciative when recruiters and vendors identify themselves through their Slack username.
```

Comment:
```suggestion
If you’re a vendor, you may advertise your product in #shameless-plugs. Advertising your product anywhere else, especially in direct messages (DMs), is strictly against the rules. We are appreciative when recruiters and vendors identify themselves through their Slack username.
```
Users in Slack forum is very wide, so be explicit about some acronyms

Reply:
Agreed! Done.